domains/cybertron.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My personal devbox for personal project",
+    "owner": {
+      "username": "varnitgoyal95",
+      "email": "varnitgoyal95@gmail.com"
+    },
+    "record": {
+      "A": ["140.238.250.204"]
```

Comment:
I cannot reach ur website

> This site can’t be reachedhttp://140.238.250.204/ is unreachable.
ERR_ADDRESS_UNREACHABLE

Reply:
its on different port try to access 

140.238.250.204:5601

Reply:
Is this domain for your AI program?